awsesproxy/Dockerfile

```diff
@@ -15,13 +15,13 @@
 ARG GO_VERSION=1.20
 FROM golang:${GO_VERSION}-alpine AS builder
 
-ENV AWSESPROXY_DIR /go/src/github.com/abutaha/aws-es-proxy
+ENV AWSESPROXY_DIR /go/src/github.com/astronomer/aws-es-proxy
 RUN set -ex \
     && apk add --no-cache make git file
 
 WORKDIR $AWSESPROXY_DIR
 
-RUN git clone  https://github.com/abutaha/aws-es-proxy.git $AWSESPROXY_DIR && git checkout v1.5 && go mod tidy
+RUN git clone  https://github.com/astronomer/aws-es-proxy.git $AWSESPROXY_DIR && git checkout a9417556b0e80196085347498b3921a92e2f85e2 && go mod tidy
```

Comment:
Having `go mod tidy` in here adds unnecessary variety to the build and potentially makes this build less reliable. Can we `go mod tidy` and commit that to our repo so we know exactly what we built with?

Reply:
i think we can remove it, let me update the commit